superset/assets/src/visualizations/PlaySlider.css

```diff
@@ -21,3 +20,7 @@
     color: #b3b3b3;
     margin-right: 5px;
 }
+
+div.tooltip.tooltip-main.top.in {
+    margin-left: 0 !important;
```

Comment:
Is there a way to override without using `!important`?

Reply:
@kristw, the 3rd-party library `bootstrap-slider` is setting the left margin directly in the DOM using `style=`, so it overrules any CSS rules. I only got it to work using `!important`.